### DISCUSSION TITLE
X11 forwarding from remote dev container

### DISCUSSION THREAD

**Discussion by jalanh11640** [devcontainers/community#178] [Q&A] [answered]
I'm trying to figure out how to configure my remote dev container to display X11 windows on my local Mac.  I ran across this [link](https://github.com/microsoft/vscode-remote-release/issues/616) which seems to indicate that an example of how to do this was added to the dev container documentation.  However, I can't seem to find the example.

Specifically, I'm looking for the example how to configure remote container (on a Linux machine) to display on a macOS machine with XQuartz running.

Thanks for any help

**Comment by chrmarti** [comment, marked as answer]
This should work out-of-the-box when `DISPLAY` is set locally for VS Code. 
Please check the Dev Containers log after connecting to the Dev Container. (`F1` > `Dev Containers: Show Container Log`)

**Reply by jalanh11640** [reply]
Thank you!  It turns out I didn’t have X11 forwarding setup in my .ssh/config
